src/Foundatio/Storage/FolderFileStorageOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Foundatio.Serializer;
+using Microsoft.Extensions.Logging;
+
+namespace Foundatio.Storage
+{
+    public class FolderFileStorageOptions
```

Comment:
We should create a base options. Seems the commonality is between the implementations is Serializer and LoggerFactory